app/controllers/application_controller.rb

```diff
@@ -11,4 +12,13 @@ def set_raven_context
                         environment: Rails.env,
                         app: ENV['VCAP_APPLICATION'])
   end
+
+  def toggle_beta_message
+    flash[:beta_message] =
+      beta_message_unseen? ? 'show' : 'hide'
```

Comment:
it seems that 'hide' is not needed as a value and could be empty

Reply:
flash[:beta_message] needs to be assigned something regardless of whether beta_message_unseen? return true or false